core/templates/components/graph-services/graph-layout.service.ts

```diff
@@ -500,15 +500,15 @@ export class StateGraphLayoutService {
 
     for (var nodeId in nodeData) {
       leftEdge = Math.min(
-        nodeData[nodeId].x0 - BORDER_PADDING, leftEdge);
+        Number(nodeData[nodeId].x0) - BORDER_PADDING, leftEdge);
```

Comment:
Why this change?

Reply:
The best case is usually to get rid of the possibility of it being `null`, the second best thing to do is to use `if`.

Reply:
ditto elsewhere